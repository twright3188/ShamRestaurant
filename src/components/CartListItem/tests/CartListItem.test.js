import React from 'react';
import renderer from 'react-test-renderer';
import { CartListItem } from '../index';

it('cart list item renders correctly', () => {
    const tree = renderer.create(<CartListItem>children</CartListItem>).toJSON();
    expect(tree).toMatchSnapshot();
});