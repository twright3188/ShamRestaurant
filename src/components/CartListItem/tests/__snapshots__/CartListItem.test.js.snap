// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`cart list item renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "backgroundColor": "#fff",
        "borderBottomColor": "rgba(34, 38, 36, 0.15)",
        "borderBottomWidth": 1,
        "flexDirection": "row",
        "height": 52,
        "paddingHorizontal": 15,
        "width": "100%",
      },
      Object {},
    ]
  }
>
  children
</View>
`;
